src/get-started/java-binding.md

```diff
@@ -523,18 +523,17 @@ For using the library just include the dependency in your `pom.xml`:
 - [Java Binding App tutorial][app-tutorial]
 
 [abstractservice]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/AbstractService.html
-[apicontrollertest]: https://github.com/exonum/exonum-java-binding/blob/v0.3/exonum-java-binding-cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java
-[app-tutorial]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding-core/rust/ejb-app/TUTORIAL.md
+[apicontrollertest]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/cryptocurrency-demo/src/test/java/com/exonum/binding/cryptocurrency/ApiControllerTest.java
+[app-tutorial]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/core/rust/ejb-app/TUTORIAL.md
 <!-- TODO: change `blob` with `0.4` after the release OR insert a Javadoc link
   instead -->
-[blockchain]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding-core/src/main/java/com/exonum/binding/blockchain/Blockchain.java
-[build-description]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding-service-archetype/src/main/resources/archetype-resources/pom.xml
+[blockchain]: https://github.com/exonum/exonum-java-binding/blob/master/exonum-java-binding/core/src/main/java/com/exonum/binding/blockchain/Blockchain.java
+[build-description]: https://github.com/exonum/exonum-java-binding/blob/v0.4/exonum-java-binding/service-archetype/src/main/resources/archetype-resources/pom.xml
 <!--TODO: Insert the correct Javadoc to Blockchain (once we publish things):
-java-binding-core/latest/com/exonum/binding/blockchain/Blockchain.html -->
-[blockchain]: https://exonum.com/doc/api/java-binding-core/latest/com/exonum/binding/service/NodeFake.html
+java-binding/core/latest/com/exonum/binding/blockchain/Blockchain.html -->
```

Comment:
As docs are published, I think we can remove this todo (and the one above for Blockchain), and put a proper link:

https://exonum.com/doc/api/java-binding-core/0.4/com/exonum/binding/blockchain/Blockchain.html